src/reaction.rs

```diff
@@ -991,42 +991,6 @@ Content-Disposition: reaction\n\
         Ok(())
     }
 
-    /// Regression test for reaction resetting self-status.
-    ///
-    /// Reactions do not contain the status,
-    /// but should not result in self-status being reset on other devices.
-    #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
-    async fn test_reaction_status_multidevice() -> Result<()> {
```

Comment:
Why the test is removed, did it break?

Reply:
It broke because it also tests that the self-status is synced from outgoing messages. Anyway, it's no longer relevant because with this PR there's nothing special to reactions anymore. The new behavior is now tested by `contact_tests::test_synchronize_status`